packages/api/cms-api/src/page-tree/blocks/internal-link.block.ts

```diff
@@ -36,9 +36,9 @@ class InternalLinkBlockData extends BlockData {
             };
         }
 
-        const readApi = pageTreeService.getReadApi({ visibility: "all" });
+        //TODO do we need createReadApi({ visibility: "all" });?
```

Comment:
I'm not sure. Using the read api from the request results in archived/unpublished page tree nodes not working in the clients. But isn't this intentional anyways?

Reply:
In theory it should be ok. I'd say as we target next branch we simply try it out